packages/rocketchat-blockstack/server/routes.js

```diff
@@ -0,0 +1,26 @@
+import { Meteor } from 'meteor/meteor';
+import { WebApp } from 'meteor/webapp';
+import { RocketChat } from 'meteor/rocketchat:lib';
+
+WebApp.connectHandlers.use('/_blockstack/manifest', Meteor.bindEnvironment(function(req, res) {
+	const name = RocketChat.settings.get('Site_Name');
+	const startUrl = Meteor.absoluteUrl();
+	const description = RocketChat.settings.get('Blockstack_Auth_Description');
+	const iconUrl = Meteor.absoluteUrl('assets/favicon_192');
```

Comment:
we should have a proper way to get assets' addresses.. [we do differently](https://github.com/RocketChat/Rocket.Chat/blob/347a26204ced260db37f21b7e6f8886323457498/packages/rocketchat-lib/server/lib/PushNotification.js#L27) on push notifications

Reply:
@sampaiodiego can you change this?